### ISSUE TITLE
Update Deployment Docs to a Linux Server

### ISSUE THREAD

**Issue by gr2m** [hoodiehq/camp#24]
This is a [starter issue](http://hood.ie/blog/starter-issues.html).
### :octocat:  Your First Pull Request?

We would like to take the opportunity to invite someone who did not yet contribute to Hoodie to work on this issue. Follow [@YourFirstPR](https://twitter.com/yourfirstPR) on Twitter for more of these issues across many open source projects :)
### 🤔 What you will need to know

We have documentation on how to deploy the old Hoodie to a linux server at https://github.com/hoodiehq/documentation/blob/gh-pages/en/deployment/linux.md. For the new "Camp Release", a lot has changed (and got simpler we hope), so the docs need to be updated. The current version is at http://docs.hood.ie/en/latest/guides/deployment.html. 
### :dart: The goal

It would be nice to have docs on how to deploy an app based on the new Hoodie to Linux. We have setup continous deployment for our Hoodie Tracker example app, docs are here: https://github.com/hoodiehq/hoodie-app-tracker/blob/master/deployment.md – maybe that’l help?
### :clipboard: Step by Step
- [x] 🙋 **Claim this issue**: Comment below (or assign yourself and continue at step 4 :)  
  Please 🙏 only claim if you want to start working on it within a day.  
  Once claimed we add you as contributor to this repository.
- [x] 👌 Accept our invitation to this repository. Once accepted, assign yourself to this repository
- [x] 👓 Please review our [Code of Conduct](http://hood.ie/code-of-conduct/)  
  In a nutshell: be patient and actively kind with us 😊
- [x] 🔄 replace the `up for grabs` label with `in progress`.
- [x] **Update the docs** :) Even if it’s work in progress, you can start a pull request for https://github.com/hoodiehq/hoodie/blob/master/docs/guides/deployment.rst. You can do that right from the browser: click on the edit icon on top right, make your changes, then at the bottom of the page click "Propose File Change" and then "Start Pull Request"
- [x] **Done** :+1: Replace the `in progress` label with `ready`. Ask in comments for a review :)

Ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/) if you have any questions :)

**Comment by distalx** [comment]
Hey I'd like to claim this issue. Thanks!

**Comment by varjmes** [comment]
@distalx issued claimed for you! :)

**Comment by gr2m** [comment]
from slack chat:

@distalx:

> RE:   `hoodiehq/camp/issues/24` https://github.com/hoodiehq/camp/issues/24 
> I’m working on this and I have a question -
> for deployment doc - (how to deploy on  linux?) 
> should I go with upstart approach or docker approach ! 
> Thanks.

**Comment by gr2m** [comment]
I’m no expert when it comes to deployment. I think we can leave out the "Deploy on hosting server" because there are no compatibale services right now, but instead add a section "Deploy with docker"?

we have a docker setup in the https://github.com/hoodiehq/hoodie-app-tracker repo, does that help?

**Comment by distalx** [comment]
@gr2m we could have different sections for deployment. like w/ or w/o docker !

**Comment by gr2m** [comment]
Yeah great idea 👍

**Comment by gr2m** [comment]
@distalx how is it going? Anything we can help with? Please don’t get stuck for too long, I know things can be very hard when working with other people’s code :) We’re happy to assist

**Comment by distalx** [comment]
@gr2m 
I think of two way when it comes to deploying hoodies to a Linux Server. 
Either it could be a step by step instructions or we could have a script which could execute all those steps behalf of the user.
I personally like the second approach So I’m playing around with the bash scripting - for last couple of days - but frankly speaking I’m not that good with the scripting - but I’m learning a lot in this process.

**Comment by gr2m** [comment]
I think both would be good? Explaining how to deploy to linux, and then offering a script for convenience?

**Comment by Enigma10** [comment]
hello ,  i have some doubts about this issue . can i use latest version of dependencies  or i have to find minimum latest version of dependencies required for it .

**Comment by gr2m** [comment]
@Enigma10 use the latest version of dependencies. Do you want to work on this issue?

**Comment by chirgjn** [comment]
@gr2m I would like to claim this. I am familiar with Linux, bash and Python but not docker, is that okay?

**Comment by gr2m** [comment]
sure :) If @Enigma10 wants to help you, you two can collaborate on it if you don’t mind?

I’ve invited you to this repository, so you can assign yourself and start checking off the list above

**Comment by chirgjn** [comment]
@gr2m I have no problem collaborating, if @Enigma10 is fine with it.
As for now I have accepted the invitation and am assigning the issue to myself and will start working on it tomorrow.

**Comment by gr2m** [comment]
@chirgjn are you still working on it?

**Comment by chirgjn** [comment]
Yes will be finishing it off this weekend.
But will need some help.
What would be a good time for a slack chat with you 😅

Sorry for the delay, some personal responsibilities required my time 😩

**Comment by gr2m** [comment]
@chrifpa I’m based in LA, best time to connect would be your morning hours if you are based in New Delhi? Tuesday would work for me

**Comment by chirgjn** [comment]
Tuesday it is then 😊

**Comment by gr2m** [comment]
@chirgjn hey sorry can we do it your Friday morning hours, my Thursday evening hours? I arrived late last night from an event

**Comment by chirgjn** [comment]
@gr2m sure. Thanks for being so helpful 😊

**Comment by chirgjn** [comment]
Hi @gr2m sorry was not available today. I won't be able to take this up until next weekend due to some work 😞. Marking it as `up for grabs` in case anybody else wants to work on this till then.

**Comment by gr2m** [comment]
Okay, no worries. I’ll be in Germany next 3 weeks, should be easier to chat, ping me on http://hood.ie/chat/ when you are around

**Comment by gr2m** [comment]
I updated the issue to reflect changes to our documentation

**Comment by gr2m** [comment]
@chirgjn still interested in working on this issue?

**Comment by sohini-roy** [comment]
Hey, I am new to hoodie and documentation is a good way to start. i would like to work on this issue. It would be a great opportunity for me. Thank you :)

**Comment by gr2m** [comment]
all yours :) I’ve invited you to this repository: https://github.com/hoodiehq/camp/invitations

Once accepted, you can start working off the checklist above :)

**Comment by sohini-roy** [comment]
Thank you :)

**Comment by sohini-roy** [comment]
Thanks again for the opportunity :)  

I have updated the document and have sent a pull request. Hope that's ok? Please have a look at it and let me know :)

Have a great weekend!

**Comment by sohini-roy** [comment]
Hello
I have updated the documentation for deploying a hoodie app to linux server. I have started a PR in the hoodie repo. Any changes needed or suggestions are welcome. :)

**Comment by sohini-roy** [comment]
Fixed the syntax and changed the mentioned lines.Please have a look? :)

**Comment by gr2m** [comment]
this is done now :) http://docs.hood.ie/en/latest/guides/deployment.html#deployment-in-linux
